cvat-ui/src/components/annotation-page/standard-workspace/image-processing.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, {
+    useCallback, useEffect, useRef,
+} from 'react';
+import { Canvas } from 'cvat-canvas-wrapper';
+import { useSelector, useDispatch } from 'react-redux';
+import { CombinedState } from 'reducers';
+import notification from 'antd/lib/notification';
+import { changeFrameAsync } from 'actions/annotation-actions';
+import { ImageFilter } from 'utils/image-processing';
+
+export default function ImageProcessingComponent(): JSX.Element | null {
```

Comment:
I believe it needs to be handled in ``cvat_enterprise/cvat/cvat-ui/src/components/annotation-page/canvas/views/canvas2d/canvas-wrapper.tsx``, passing to canvas correctly setup Proxy object instead of FrameData instance.